docs/contributor/arch/011-kymastatsmetrics.md

```diff
@@ -0,0 +1,116 @@
+Another available source to monitor changes of arbitrary Kubernetes objects is the [Kubernetes Objects Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/k8sobjectsreceiver). This receiver produces only logs and no metrics.
+
+The OpenTelemetry Collector provides interfaces for enhancements by implementing custom receiver plugins. The OpenTelemetry project provides [documentation](https://opentelemetry.io/docs/collector/building/receiver/) on implementing a custom receiver and adding it to a custom distribution of the Collector.
+
+## Decision
+
+Due to the restrictions of available telemetry resources for Kubernetes resources, building a custom receiver is the most suitable option.
+
+### Extended MetricPipeline API
+
+Module status metrics can be activated as a new input in the MetricPipeline CRD. Users can select the Kyma modules of interest by giving a module list. An empty list will include metrics of all active modules.
+
+```yaml
+apiVersion: telemetry.kyma-project.io/v1alpha1
+kind: MetricPipeline
+metadata:
+  name: sample
+spec:
+  input:
+    kyma:
+      enabled: true
+      modules:
+        - telemetry
+    runtime:
+      enabled: true
+    prometheus:
+      enabled: true
+    istio:
+      enabled: true
+  output:
+    otlp:
+      endpoint:
+        value: http://example.com:4317
+```
+
+Enabling the Kyma input will enable a custom metrics receiver, called `kymastats`, in the telemetry-metric-gateway that produces metrics for the module state and status conditions.
+The receiver configuration will follow the shown example.
+
+```yaml
+receivers:
+  kymastats:
+    k8s_cluster:
+        auth_type: serviceAccount
+    collection_interval: 30s
+    api_groups:
+    - operator.kyma-project.io
+```
+
+The receiver needs the following properties:
+
+- **auth_type**: The way to authenticate with the Kubernetes API. Possible values are `serviceAccount` (default) or `kubeConfig`.
+- **collection_interval**: The interval that is used to emit metrics.
+- **api_groups**: List of API groups to scrape. Most of the Kyma modules should use the `operator.kyma-project.io` API group. The list can be extended to support monitoring custom modules. Every CRD in the listed groups is assumed to represent a module.
+
+### Custom Metrics Receiver for OpenTelemetry Collector
+
+We assume the status subresource of a module CRD to contain a `conditions` list that uses the [meta/v1/Condition](https://pkg.go.dev/k8s.io/apimachinery@v0.30.0/pkg/apis/meta/v1#Condition) type and an overarching state attribute. We assume positive polarity for all conditions.
+
+An example for this structure is the status subresource of the Telemetry module:
```

Comment:
I assume we should format it as [parameter](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#use-code-font-for-these-items)? 

```suggestion
As an example for this structure, see the **status** subresource of the Telemetry module:
```

Reply:
The shown yaml is generated by the operator. So rather not a parameter.

Reply:
Then is it a [code example](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/03-formatting.md#items-written-in-code-font)?

```suggestion
As an example for this structure, see the `status` subresource of the Telemetry module:
```

Reply:
I think that fits better according to the table in the guide. Changed it.